[WIP] Polish offline UI to match mockup

deeplow opened the pull request:
WIP: 
- ~does not decrease opacity of left pane logo when starting out in offline mode~
- tweak gradient

# Description
- reverse gradient in top panel of main window (#583 for explanation)
- change sd logo opacity based on offline/online mode (#583 for explanation)
- small fix #585
- ~small fix #584~ (it may be better to leave this for another PR)

Fixes #583, #585, ~#584~


## Visual changes summary

SD logo on left panel now changes according to online/offline mode:

online: ![top0](https://user-images.githubusercontent.com/47065258/67465124-1dc9af80-f634-11e9-9228-a164089bc014.png)
offline: ![top1](https://user-images.githubusercontent.com/47065258/67465139-228e6380-f634-11e9-80ce-42fae1d82b4a.png)


Gradient on top panel now changes according to online/offline mode:

| online | offline |
|---------|---------|
|![commit-pane-online](https://user-images.githubusercontent.com/47065258/67463302-b3fbd680-f630-11e9-9f7e-db05b6dcdd76.png) | ![commit-pane-offline](https://user-images.githubusercontent.com/47065258/67463310-b5c59a00-f630-11e9-848e-9c2fd277e060.png)|

**note:** on the offline left pane the *sign in* button is cropped. That is #584 which is left unaddressed in this PR

# Test Plan
1. open in online mode
2. confirm the logo on the left pane has high opacity
3. confirm the top pane has right gradient for online mode
4. logout
5. confirm the logo on the left pane has now low opacity
6. confirm the top pane has [right gradient](https://zpl.io/Vqv0vwJ) for offline mode

# Checklist
If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:

 - [ ] I have tested these changes in the appropriate Qubes environment
 - [ ] I do not have an appropriate Qubes OS workstation set up (the reviewer will need to test these changes)
 - [x] These changes should not need testing in Qubes

deeplow commented:
I actually preferred the way the gradient on the top bar looked before.

Before:

![gradient2](https://user-images.githubusercontent.com/47065258/67395120-e5bf5f80-f594-11e9-97d3-a6e6a5c21500.png)


After:

![gradient1](https://user-images.githubusercontent.com/47065258/67395105-e22bd880-f594-11e9-9553-1ee61639d6dc.png)

I guess it has to do with a very high contrast between the light blue and the dark blue on the top left corner.

ninavizz commented:
The gradient is not supposed to exactly be "reversed,; I think it's a fully different gradient. Zeplin shows the correct one. Agreed, I also prefer the "online" bar, but felt it's important for there to be a clear visual distinction beyond the different icon used. And, to be clear—this PR is _just_ for the gradient, with an understanding that in offline mode there's also a different icon and text styling in the refresh widget?

The _Branding Bar_ is also supposed to have lower contrast artwork in it... which I don't think Allie had yet implemented. Updated art for that, I'll be generating towards the end of this week, or early next week. 

_Offline Mode_ should not be ugly, but it should be a little visually different from _Online Mode_, following the metaphor that "things are plugged-in!" when in `online mode`, and "things have been shut-off for the night" in `offline mode`.

deeplow commented:
> The gradient is not supposed to exactly be "reversed,; I think it's a fully different gradient. Zeplin shows the correct one. Agreed, I also prefer the "online" bar, but felt it's important for there to be a clear visual distinction beyond the different icon used

You're totally right! I missed the part where there would be a gradient for the offline mode an one for the online mode. Just disregard my comment on that. I also noticed that I made an even gradient and the mockup describes an uneven one. Will fix that now

> And, to be clear—this PR is _just_ for the gradient, with an understanding that in offline mode there's also a different icon and text styling in the refresh widget?

No. In this PR I did:
> - reverse gradient in top panel of main window (#583 for explanation)
> - branding bar opacity based on offline/online mode (#583 for explanation)
> - small fix #585
> - small fix #584

I forgot to change the offline icon in the top bar though, will fix that now. I also tackled very minor issues that I go in the way and had to do with the code I was overwriting (#585, #584), if that is ok.

deeplow commented on securedrop_client/gui/widgets.py:
```diff
@@ -104,9 +113,17 @@ def setup(self, controller):
 
     def enable_refresh(self):
         self.refresh.enable()
+        self.set_online_style()
```
`enable_refresh` was the function triggered when the offline/online mode changes. By making this change to the style and adding it to that function it makes it go beyond its name. So I would propose changing either to change the name to `set_online` (and propagate changes to `gui/main.py`. But I wanted feedback before doing that.

redshiftzero replied:
I dig it, `set_online` seems like a clear name to me

deeplow commented:
- reworked the commits so they are more contained and improved the code
- implemented top panel gradient change with online/offline mode

Visual description of the PR [added in first comment](https://github.com/freedomofpress/securedrop-client/pull/586#issue-331520115)

creviera commented:
Tested in qubes and i see the top bar gradient changes now when switching between offline and online. I like the feel of going dark when offline. (Also, just a note that the `SIGN IN` button looks centered in qubes, but from your screenshot I see that it's totally out of alignment.)

It would look better if, when switching to online mode, the left pane didn't have to repaint, but that is something that was happening before. 

lgtm!
